content/resources/titlebar.js

```diff
@@ -59,7 +59,7 @@ function setupMenu (node, meta) {
     groupsObj[name].forEach(function (module) {
       groupModulesContainer.add(dx.detached('a')
         .class('docs-dropdown-module-link dx-section dx-fixed')
-        .attr('href', '/docs/' + (meta.selectedVersion || meta.targetVersions[meta.targetVersions.length - 1]) + '/' + module + '/')
+        .attr('href', '/docs/' + (meta.selectedVersion ? meta.selectedVersion + '/' : '') + module + '/')
```

Comment:
By default, the menu will navigate to `docs/<module>` unless a version is present in the URL